src/_version.ts

```diff
@@ -0,0 +1,6 @@
+// NOTE: should be synced with package.json
+
+/**
+ *  The current version of the Relayer SDK.
+ */
+export const version: string = '0.1.0-1';
```

Comment:
You should use version from package.json

Move this file in `/` directory and write

```
import package from './package.json';
/**
 *  The current version of the Relayer SDK.
 */
export const version: string = package.version;
```